webapp/src/components/training/TrainingSteps.vue

```diff
@@ -2,16 +2,16 @@
   <div v-if="task !== undefined">
     <Description v-show="trainingStore.step === 1" :task="task" />
 
-    <div
+    <divlabeled_dataset
```

Comment:
ho, what is that? vue gives me a warning about it.
```
[Vue warn]: Failed to resolve component: divlabeled_dataset
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <TrainingSteps id="tinder_dog" onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< 
Proxy { <target>: Proxy, <handler>: {…} }
 >  ... > 
  at <KeepAlive> 
  at <RouterView> 
  at <BaseLayout> 
  at <App>
```

```suggestion
    <div
```